.github/workflows/linux.yml

```diff
@@ -215,13 +199,7 @@ jobs:
         run: |
           set -e # Instead of trying to chain everything together with &&
```

Comment:
This and `set +e` are not needed any more as there is now a single command. Please remove.